### PR TITLE
Fix logic to stop app

### PR DESCRIPTION
**Pull request by obino**

### CODE REVIEW

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/2132/

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/2134/

File: AppController/djinn.rb [hunk 1]
```diff
@@ -1635,10 +1635,10 @@ def stop_app(app_name, secret)
         @app_names << "none" if @app_names.empty?
       }
 
-      # Prevent future deploys from using the old application code.
-      FileUtils.rm_rf("#{HelperFunctions.get_app_path(app_name)}")
-      FileUtils.rm_rf("#{PERSISTENT_MOUNT_POINT}/apps/#{app_name}.tar.gz")
-      CronHelper.clear_app_crontab(app_name)
+      # To prevent future deploys from using the old application code, we
+      # forced a removal of the application status on disk (for example
```
Comment (menivaitsi) [inline AppController/djinn.rb]: forced -> force

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/2135/

**Pull request merged**